DefaultModules.py

```diff
@@ -1,634 +1,640 @@
+                x=bottom_left_position[0],
+                y=bottom_left_position[1] - size[1],
+                width=size[0],
+                height=size[1],
+            )
+        elif entity.dxftype() == "MTEXT":
+            insert = entity.dxf.insert
+            element = SVGText(x=insert[0], y=insert[1], text=entity.text)
+        elif entity.dxftype() == "TEXT":
+            insert = entity.dxf.insert
+            element = SVGText(x=insert[0], y=insert[1], text=entity.dxf.text)
+        elif entity.dxftype() == "SOLID" or entity.dxftype() == "TRACE":
+            # https://ezdxf.readthedocs.io/en/stable/dxfentities/solid.html
+            element = Path()
+            element.move((entity[0][0], entity[0][1]))
+            element.line((entity[1][0], entity[1][1]))
+            element.line((entity[2][0], entity[2][1]))
+            element.line((entity[3][0], entity[3][1]))
+            element.closed()
+            element.fill = Color("Black")
+        elif entity.dxftype() == "SPLINE":
+            element = Path()
+            try:
+                for b in entity.construction_tool().bezier_decomposition():
+                    if len(element) == 0:
+                        element.move((b[0][0], b[0][1]))
+                    if len(b) == 4:
+                        element.cubic(
+                            (b[1][0], b[1][1]), (b[2][0], b[2][1]), (b[3][0], b[3][1])
+                        )
+                    elif len(b) == 3:
+                        element.quad((b[1][0], b[1][1]), (b[2][0], b[2][1]))
+            except (AttributeError, TypeError):
+                # Fallback for rational b-splines.
+                try:
+                    for bezier in entity.construction_tool().cubic_bezier_approximation(
+                        4
+                    ):
+                        b = bezier.control_points
+                        if len(b) == 4:
+                            element.cubic(
+                                (b[1][0], b[1][1]),
+                                (b[2][0], b[2][1]),
+                                (b[3][0], b[3][1]),
+                            )
+                        elif len(b) == 3:
+                            element.quad((b[1][0], b[1][1]), (b[2][0], b[2][1]))
+                except (AttributeError, TypeError):
+                    # Fallback for versions of EZDXF prior to 0.13
+                    element.move(entity.control_points[0])
+                    for i in range(1, entity.dxf.n_control_points):
+                        element.line(entity.control_points[i])
+            if entity.closed:
+                element.closed()
+        elif entity.dxftype() == "INSERT":
+            for e in entity.virtual_entities():
+                if e is None:
+                    continue
+                DxfLoader.entity_to_svg(elements, dxf, e, scale, translate_y)
+            return
+        else:
+            return  # Might be something unsupported.
+
+        try:

```

Comment:
This marks the start of the actual change. The import is in a try/except. These items changed locations in ezdxf 0.6.15+